mon-pix/mirage/config.js

```diff
@@ -50,6 +50,8 @@ export default function() {
   this.post('/assessments', postAssessments);
   this.get('/assessments/:id', getAssessment);
   this.get('/assessments/:assessmentId/next', getNextChallenge);
+  this.get('/assessments', findAssessments);
+  this.patch('/assessments/:id/complete-assessment', () => null, 204);
```

Comment:
il me semble pas que la route renvoie null avec un code 204. De plus, la spec `json:api` dit `if a server accepts an update but also changes the resource(s) in ways other than those specified by the request (for example, updating the updated-at attribute or a computed sha), it MUST return a 200 OK response.` Nous derrière on créée une entité entière donc pas de 204, mais la réponse est plutôt 200.